client/landing/browsehappy/index.jsx

```diff
@@ -8,7 +9,11 @@ import './style.scss';
 
 const SUPPORTED_BROWSERS_LINK = 'https://wordpress.com/support/browser-issues/#supported-browsers';
 
-export default function Browsehappy() {
+export default function Browsehappy( { from } ) {
+	// `from` is passed into the component via server-side-render. Since the query
+	// param is sanitized and validated on the server, we can trust it here.
+	const continueUrl = addQueryArgs( { bypassTargetRedirection: true }, from );
```

Comment:
I turns out that checking `from` is not needed here, because it's guaranteed to be provided by the route handler. The `/` defaulting is performed there.